src/crate/client/doctests/client.txt

```diff
@@ -213,7 +213,7 @@ supported, all other fields are 'None'::
     >>> pprint(result)
     ['Aldebaran',
      1373932800000,
-     None,
+     1373932800000,
```

Comment:
maybe you can add also time info, down to millis?

Reply:
Sure. a7fa1ee55f brings in a corresponding improvement.

Reply:
As discussed, after that we need to add a column of `DATE` to test table `locations`, and we need also to add functionality to the client to truncate to start of day when using python date type, even if the table column (result) is a complete timestamp with time info.